content/docs/next-release/opentelemetry.md

```diff
@@ -0,0 +1,114 @@
+
+Component             | Repository
+--------------------- | ---
+**jaeger-agent**      | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-agent/)
+**jaeger-collector**  | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-collector/)
+**jaeger-ingester**   | [hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/](https://hub.docker.com/r/jaegertracing/jaeger-opentelemetry-ingester/)
+**all-in-one**        | [hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/](https://hub.docker.com/r/jaegertracing/opentelemetry-all-in-one/)
+
+## Compatibility
+
+The Jaeger OpenTelemetry components are **almost** backward compatible legacy Jaeger components.
+
+The differences are:
+
+* Health check port changed to `13133`
+* Not all legacy Jaeger flags are exposed (e.g. health check port)
+* Exposed metrics
+
+## Configuration
+
+Jaeger OpenTelemetry components can be configured by a subset of Jaeger legacy flags (or other [configuration sources](deployment.md#configuration-3))
+and [OpenTelemetry configuration file](https://opentelemetry.io/docs/collector/configuration/).
+The OpenTelemetry configuration takes precedence over Jaeger configuration.
+The Jaeger OpenTelemetry components use hardcoded default configuration that enables predefined set of components - Jaeger receiver, attribute processor, (storage) exporter.
+The opinionated default configuration ensures compatibility between Jaeger legacy components.
+The user provided OpenTelemetry configuration is merged with the default configuration.
+
+Let's have a look at example configuration:
+
+```sh
+$ docker run --rm -it -v ${PWD}:/config \
+    -e SPAN_STORAGE_TYPE=elasticsearch \
+    jaegertracing/jaeger-opentelemetry-collector \
+    --config-file=/config/config.yaml \
+    --es.server.urls=http://localhost:9200 \
+    --es.num-shards=3
+```
+
+The content of `config.yaml`:
+
+```yaml
+exporters:
+  jaeger_elasticsearch:
+    es:
+      server-urls: http://elasticsearch:9200
+      num-replicas: 2
+processors:
+  attributes:
+    actions:
+      - key: user
+        action: delete
+service:
+  pipelines:
+    traces:
+      processors: [attributes]
+```
+
+* Enables Jaeger receiver with collector endpoints - gRPC, TChannel, HTTP. The Jaeger receiver is enabled by default.
+* Enables Elasticsearch backend with URL http://elasticsearch:9200, 3 primary shards (default is 5) and 2 replica shards (default 1)
+* Enables attribute processor (disabled by default). Note that new components have to be explicitly added to the pipeline.
+* Enables health check extension.
+
+The following command can be used to list all supported flags:
+
+```sh
+$ docker run --rm \
+    -e SPAN_STORAGE_TYPE=elasticsearch \
+    jaegertracing/jaeger-opentelemetry-collector:latest -h \
+```
+
+## Jaeger Operator
```

Comment:
Is "Operator" the right title here? Seems like it's a manual CRD.

Reply:
Yes it's the right title.